Cactooz/reShuffle
# Shuffles

### Completed tasks
- Add fetch functions for fetching songs from a playlist and audio features of songs
- Add so that the playPlaylist function sends songs to the playback queue according to the chosen shuffle function
- Remove unused playlist context uri
- Add shuffles:
  0. Spotify 2014 Shuffle
  - 1, Fisher Yates Shuffle
  - 2, Epic shuffle version 1, the distance shuffle (Name can be changed in future)
  - Default, if any other shuffle button is pressed, Fisher Yates Shuffle
- These are the first versions of the different shuffles and can change and get optimized with time

### Testing
1. Go to reShuffle and log in
2. Press play on any playlist, it should start playing on an active device

### Bugs
Noticed when I launched reShuffle without any other active devices it failed to find an active device sometimes. Like if you quickly switch between playlists.  It will give you a 404 in the console. But then when you do it the second time it works. It seems like you get a new device id every time you do a new shuffle. If you have another device open at the same time and play on it but still use reShuffle to switch between playlists, it works fine.
